config/config_gaea.sh

```diff
@@ -3,12 +3,13 @@
 # Compiler/MPI combination
 export HPC_COMPILER="intel/18.0.6.288"
 export HPC_MPI="cray-mpich/7.7.11"
+export HPC_PYTHON="cray-python/3.7.3.2"
 
 # Build options
 export USE_SUDO=N
 export PKGDIR=pkg
 export LOGDIR=log
-export OVERWRITE=N
+export OVERWRITE=Y
```

Comment:
Is this meant to be on?

Reply:
As long as I maintain the stack on gaea, yes please. It's on for Cheyenne, too.